front-end/src/components/Banner/Banner.jsx

```diff
@@ -0,0 +1,23 @@
+import './Banner.css';  
+import { useState } from 'react';    
+import { banner112022, banner042023, banner112023 , banner102024 } from '../../assets/banners';
```

Comment:
should write a util to read file from a path and return to use rather than import and export in js file